app/components/DocumentBreadcrumb.tsx

```diff
@@ -69,7 +69,7 @@ const DocumentBreadcrumb: React.FC<Props> = ({
     : undefined;
 
   React.useEffect(() => {
-    void document.loadRelations();
+    void document.loadRelations({ withoutPolicies: true });
```

Comment:
In what situation are we displaying a document breadcrumb but we don't need the document policies? 🤔

Reply:
I would ask the opposite – in what situation do we need the document policies for displaying a document breadcrumb? 😄

Reply:
Don't have a solution here yet but looks like this is an existing bug. I mean, even after loading policies, we don't really use those to _decide_ what the breadcrumb should look like.